example/feature_demo/demo_service.pb.gorm.go

```diff
@@ -190,54 +194,102 @@ func DefaultStrictUpdateIntPoint(ctx context.Context, in *IntPoint, db *gorm.DB)
 type IntPointDefaultServer struct {
 	DB *gorm.DB
 }
+type IntPointCreateCustomHandler interface {
+	CustomCreate(context.Context, *CreateIntPointRequest) (*CreateIntPointResponse, error)
+}
 
 // Create ...
 func (m *IntPointDefaultServer) Create(ctx context.Context, in *CreateIntPointRequest) (*CreateIntPointResponse, error) {
+	if custom, ok := interface{}(m).(IntPointCreateCustomHandler); ok {
+		return custom.CustomCreate(ctx, in)
+	}
 	res, err := DefaultCreateIntPoint(ctx, in.GetPayload(), m.DB)
 	if err != nil {
 		return nil, err
 	}
 	return &CreateIntPointResponse{Result: res}, nil
 }
 
+type IntPointReadCustomHandler interface {
+	CustomRead(context.Context, *ReadIntPointRequest) (*ReadIntPointResponse, error)
+}
+
 // Read ...
 func (m *IntPointDefaultServer) Read(ctx context.Context, in *ReadIntPointRequest) (*ReadIntPointResponse, error) {
+	if custom, ok := interface{}(m).(IntPointReadCustomHandler); ok {
```

Comment:
why does it need to be wrapped in the `interface{}(m)` instead of just `m.(Foo)`?

Reply:
Otherwise this happens
```
example/feature_demo/demo_service.pb.gorm.go:218: invalid type assertion: m.(IntPointReadCustomHandler) (non-interface type *IntPointDefaultServer on left)
```

Reply:
I still don't understand Go 🤷‍♂️